### ISSUE TITLE
Uninstall must remove PostgreSQL backports from apt sources

### ISSUE THREAD

**Issue by joel-coffman** [USAFA-CompSci364/vm#25]
Remove `/etc/apt/sources.list.d/pgdb.list` lest the sources cause issues when trying to install additional software (including the postgresql-common package).

**Comment by danieljeong24** [comment]
Example of error when installing software without first removing the backport sources:

~~~ text
Hit:1 http://us.archive.ubuntu.com/ubuntu bionic InRelease
Get:2 http://us.archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]
Get:3 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]
Get:4 http://us.archive.ubuntu.com/ubuntu bionic-backports InRelease [83.3 kB]
Get:5 http://security.ubuntu.com/ubuntu bionic-security/main amd64 DEP-11 Metadata [76.8 kB]
Get:6 http://us.archive.ubuntu.com/ubuntu bionic-updates/main i386 Packages [1,604 kB]
Get:7 http://security.ubuntu.com/ubuntu bionic-security/universe amd64 DEP-11 Metadata [61.0 kB]
Get:8 http://security.ubuntu.com/ubuntu bionic-security/multiverse amd64 DEP-11 Metadata [2,464 B]
Get:9 https://apt.postgresql.org/pub/repos/apt bionic-pgdg InRelease [116 kB]
Err:9 https://apt.postgresql.org/pub/repos/apt bionic-pgdg InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 7FCC7D46ACCC4CF8
Get:10 http://us.archive.ubuntu.com/ubuntu bionic-updates/main amd64 Packages [2,892 kB]
Get:11 http://us.archive.ubuntu.com/ubuntu bionic-updates/main amd64 DEP-11 Metadata [296 kB]
Get:12 http://us.archive.ubuntu.com/ubuntu bionic-updates/universe amd64 DEP-11 Metadata [302 kB]
Get:13 http://us.archive.ubuntu.com/ubuntu bionic-updates/multiverse amd64 DEP-11 Metadata [2,468 B]
Get:14 http://us.archive.ubuntu.com/ubuntu bionic-backports/main amd64 DEP-11 Metadata [8,124 B]
Get:15 http://us.archive.ubuntu.com/ubuntu bionic-backports/universe amd64 DEP-11 Metadata [10.0 kB]
Fetched 5,515 kB in 2s (2,685 kB/s)
Reading package lists...
Building dependency tree...
Reading state information...
244 packages can be upgraded. Run 'apt list --upgradable' to see them.
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://apt.postgresql.org/pub/repos/apt bionic-pgdg InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 7FCC7D46ACCC4CF8
W: Failed to fetch https://apt.postgresql.org/pub/repos/apt/dists/bionic-pgdg/InRelease  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 7FCC7D46ACCC4CF8
W: Some index files failed to download. They have been ignored, or old ones used instead.
~~~

**Comment by joel-coffman** [comment]
Potentially resolved by e365440, but no longer directly relevant because a PostgreSQL backport is not being installed (#26).
